### PR TITLE
Adding Authors for Methylation Section

### PR DESCRIPTION
**Pull request by jlevy44**
Hi there! 

Just adding our author information.

Please, @AlexanderTitus and @brockclarke can you verify that these details are correct? Thanks!

### CODE REVIEW

File: content/metadata.yaml [hunk 1]
```diff
@@ -70,6 +70,39 @@ authors:
...
+    name: Joshua J. Levy
+    orcid: 0000-0001-8050-1291
+    email: joshua.j.levy.gr@dartmouth.edu
+    affiliations:
+      - Program in Quantitative Biomedical Sciences, Geisel School of Medicine at Dartmouth, Lebanon, NH
+    funders: Burroughs Wellcome Fund Big Data in the Life Sciences training grant at Dartmouth
+    v1: "Drafted one or more subsections"
+    v2: ""
+    coi:
+      string: "None"
```
Comment (cgreene) [inline content/metadata.yaml]: Can you also add a last approved date? See the other entries. Can make it today.

File: content/metadata.yaml [hunk 2]
```diff
@@ -70,6 +70,39 @@ authors:
...
       string: "None"
       lastapproved: !!str 2017-05-26
+  - github: jlevy44
+    name: Joshua J. Levy
+    orcid: 0000-0001-8050-1291
+    email: joshua.j.levy.gr@dartmouth.edu
+    affiliations:
+      - Program in Quantitative Biomedical Sciences, Geisel School of Medicine at Dartmouth, Lebanon, NH
+    funders: Burroughs Wellcome Fund Big Data in the Life Sciences training grant at Dartmouth
+    v1: "Drafted one or more subsections"
```
Comment (cgreene) [inline content/metadata.yaml]: Can you switch these from v1 which should be an empty string to v2?

**Comment by brockclarke** [comment]
I approve, and have no conflicts.
The funding information provided did not appear to populate in the draft build.
Thanks

**Comment by cgreene** [comment]
@brockclarke funding is not yet automated. I can try to do that on a future PR. Thanks for noting that! I've created #996 to address this.

File: content/metadata.yaml (hunk 2 above)
Comment (jlevy44) [reply content/metadata.yaml]: Okay I tried to give it a shot. I wasn't sure if the date was supposed to be when our PRs were merged.

File: content/metadata.yaml (hunk 1 above)
Comment (jlevy44) [reply content/metadata.yaml]: Okay I tried to give it a shot. I wasn't sure if the date was supposed to be when our PRs were merged.

File: content/metadata.yaml (hunk 2 above)
Comment (jlevy44) [reply content/metadata.yaml]: Sure, just made the change.

File: content/metadata.yaml (hunk 1 above)
Comment (cgreene) [reply content/metadata.yaml]: The date should be when you last checked the COI information. I think it should be when Brock commented and whenever you last checked it. Can you prod @AlexanderTitus to give a 👍 on inclusion and confirm/approve COI info as correct?

File: content/metadata.yaml [hunk 3]
```diff
@@ -70,6 +70,42 @@ authors:
...
+    funders: NIH R01CA216265, Dartmouth College Neukom Institute for Computational Science CompX award
+    v1: ""
+    v2: "Drafted one or more subsections"
+    coi:
+      string: "None"
+      lastapproved: !!str 2019-08-01
+  - github: AlexanderTitus
+    name: Alexander J. Titus
+    orcid: 0000-0002-0145-9564
+    email: alexander.j.titus@gmail.com
```
Comment (AlexanderTitus) [inline content/metadata.yaml]: @jlevy44 can you please update my email to Alexander.J.Titus@Dartmouth.edu. 

After that, I approve the update, and confirm that I have COI is "none". 

Thanks all.

Comment (jlevy44) [reply content/metadata.yaml]: Done!

**Comment by cgreene** [comment]
Assuming this build passes (so, if I didn't botch something) I'll get this merged in!

**Comment by cgreene** [comment]
Woo! Congrats 🎊  - welcome to the team and author list for v2.0 :)

**Pull request merged**